doc/policies_list/3.10.0/policies.json

```diff
@@ -0,0 +1,2718 @@
+        "summary": "Custom metrics on Nginx post actions ",
+        "description": [
+          "With this policy, on post_actions the Authrep call will report any new ",
+          "metric if one of the conditions match. The main use case for this is to ",
+          "report any metric based on response headers, status codes, or any other ",
+          "liquid exposed variable."
+        ],
+        "order": {
+          "before": [
+            {
+              "version": "builtin",
+              "name": "apicast"
+            }
+          ]
+        },
+        "$schema": "http://apicast.io/policy-v1.1/schema#manifest#",
+        "configuration": {
+          "definitions": {
+            "operation": {
+              "required": [
+                "left",
+                "op",
+                "right"
+              ],
+              "properties": {
+                "op": {
+                  "enum": [
+                    "==",
+                    "!=",
+                    "matches"
+                  ],
+                  "description": "Operation to apply. The matches op supports PCRE (Perl compatible regular expressions)",
+                  "type": "string"
+                },
+                "right": {
+                  "type": "string"
+                },
+                "left_type": {
+                  "oneOf": [
+                    {
+                      "title": "Evaluate 'left' as plain text.",
+                      "enum": [
+                        "plain"
+                      ]
+                    },
+                    {
+                      "title": "Evaluate 'left' as liquid.",
+                      "enum": [
+                        "liquid"
+                      ]
+                    }
+                  ],
+                  "description": "How to evaluate 'left'",
+                  "default": "plain",
+                  "type": "string"
+                },
+                "left": {
+                  "type": "string"
+                },
+                "right_type": {
+                  "oneOf": [
+                    {
+                      "title": "Evaluate 'right' as plain text.",
+                      "enum": [
+                        "plain"
+                      ]
+                    },
+                    {
+                      "title": "Evaluate 'right' as liquid.",
+                      "enum": [
+                        "liquid"
+                      ]
+                    }
+                  ],
+                  "description": "How to evaluate 'right'",
+                  "default": "plain",
+                  "type": "string"
+                }
+              },
+              "type": "object",
+              "$id": "#/definitions/operation"
+            },
+            "custom_metrics_rule": {
+              "properties": {
+                "increment": {
+                  "description": "How many hits should be incremented, liquid value ",
```

Comment:
I'm not sure what this description is trying to say. Is it: `This metric determines how many hits should be incremented using the liquid value`?